tests/language_specific/english/test_english_language_generator.py

```diff
@@ -70,6 +71,29 @@ def test_simple_verb():
     ).as_token_sequence() == ("Mom", "pushes", "a", "table")
 
 
+def test_simple_quantifier():
```

Comment:
Also add a test with just one box. Also maybe rename this test to `test_two_objects` since it isn't clear to the reader what "simple" means in this context"

Reply:
I thought adding a test with one box would have been repetitive since we already have `test_common_noun`.